frontend/src/components/worksheets/InformationModal/InformationModal.js

```diff
@@ -45,13 +45,13 @@ const KeyboardShortcutModal = ({ show, toggle }) => (
                     </tr>
                     <tr>
                         <td>
-                            <kbd>enter</kbd>
+                            <kbd>shift+enter</kbd>
```

Comment:
This isn't correct. enter is still a keyboard shortcut for "start editing markdown".

You need to add a new entry for "shift+enter" that means "Add a new line in markdown".